contracts/airdrop.sol

```diff
@@ -62,7 +62,11 @@ contract Airdrop {
         if (!available) {
             claimable = 0;
         } else if (block.timestamp > info.start_time) {
-            claimable = amount - amount * (((block.timestamp - info.start_time) / 86400) * 2) / 10;
+            if ((block.timestamp - info.start_time) / 86400 > 5) {
```

Comment:
SafeMath doesn't take effect, after 6 days after start_time, overflow.

```
AssertionError: expected '115792089237316195423570985008687907853269984665640564039457584007913129631827' to equal '0'
```

Reply:
Since here info.start_time is uint48. I will fix this.